.github/workflows/build-publish.yml

```diff
@@ -16,19 +16,19 @@ jobs:
   natives-ios:
     runs-on: macos-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```

Comment:
Berstanio was against this when I mentioned failing builds a couple months ago.

> No, v4 requires node 20, while v3 is fine with 16

But whether that's still the case and if we need old versions for anything, I do not know.